.lintrc

```diff
@@ -1,6 +1,7 @@
 # See https://github.com/markstory/lint-review
 [files]
 ignore = */migrations/*
```

Comment:
i believe if we use the django commands (as opposed to the south commands) it will create the migration in the `migrations` folder. though emord can confirm